plugins/LadspaEffect/swh/CMakeLists.txt

```diff
@@ -4,13 +4,16 @@ INCLUDE_DIRECTORIES("${CMAKE_SOURCE_DIR}/include"
                     ${FFTW3F_INCLUDE_DIRS}
                     "${CMAKE_BINARY_DIR}")
 LINK_DIRECTORIES(${FFTW3F_LIBRARY_DIRS})
-LINK_LIBRARIES(-lfftw3f)
 FILE(GLOB PLUGIN_SOURCES "${CMAKE_CURRENT_SOURCE_DIR}/*.c")
 
 FOREACH(_item ${PLUGIN_SOURCES})
 	GET_FILENAME_COMPONENT(_plugin "${_item}" NAME_WE)
 
 	ADD_LIBRARY("${_plugin}" MODULE "${_item}")
+	# vocoder_1337 does not use fftw3f
+	IF(NOT ("${_plugin}" STREQUAL "vocoder_1337"))
+		TARGET_LINK_LIBRARIES("${_plugin}" -lfftw3f)
+	ENDIF()
```

Comment:
Good catch!  We can merge this, but it also needs to make it to https://github.com/swh/ladspa.  Many projects use Steve's upstream `swh` LADSPA packages and Steve's pretty good about merging in these types of PRs.